docs/faq.md

```diff
@@ -83,8 +83,39 @@ We list some common troubles faced by many users and their corresponding solutio
     1. This error indicates that your module has parameters that were not used in producing loss. This phenomenon may be caused by running different branches in your code in DDP mode.
     2. You can set ` find_unused_parameters = True` in the config to solve the above problems or find those unused parameters manually.
 
+- Save the best model
+
+    It can be turned on by configuring `evaluation = dict(save_best=‘auto’)`. In the case of the `auto` parameter, the first key in the returned evaluation result will be used as the basis for selecting the best model. You can also directly set the key in the evaluation result to manually set it, for example, `evaluation = dict(save_best='mAP' )`.
+
+- Usage of EMA Hook in Resume
+
+    If you use `EMA Hook` in training, you can't use command line parameters to restore model parameters during resume, you can only do so by modifying the `resume_from` field in the configuration file. This is because the implementation of `EMA Hook` is quite special, and the weights need to be loaded again during initialization.
+
 ## Evaluation
 
 - COCO Dataset, AP or AR = -1
     1. According to the definition of COCO dataset, the small and medium areas in an image are less than 1024 (32\*32), 9216 (96\*96), respectively.
     2. If the corresponding area has no object, the result of AP and AR will set to -1.
+
+## Model
+
+- ResNet style parameter description
+
+    ResNet `style` optional parameters include `pytorch` and `caffe`, the difference lies in the Bottleneck module. Bottleneck is a stacking structure of `1x1-3x3-1x1`. In the case of sampling with stride=2, the stride parameter of `caffe` mode is placed on the first `1x1` convolution, while in `pyorch` mode, stride is placed on the second `3x3` convolution. The sample code is as follows:
+
+    ```
+    if self.style == 'pytorch':
+            self.conv1_stride = 1
+            self.conv2_stride = stride
+        else:
+            self.conv1_stride = stride
+            self.conv2_stride = 1
+    ```
+
+- ResNeXt parameter description
+
+    ResNeXt comes from the paper `Aggregated Residual Transformations for Deep Neural Networks`. It introduces grouped convolution and uses “cardinality” to control the number of groups to achieve a balance between accuracy and complexity. It controls the basic width and grouping parameters of the internal Bottleneck module through two hyperparameters `baseWidth` and `cardinality`. The configuration name in MMDetection is `mask_rcnn_x101_64x4d_fpn_mstrain-poly_3x_coco.py`, where `mask_rcnn` represents the algorithm using Mask R-CNN, `x101` represents the backbone network using ResNeXt-101, and `64x4d` represents the Botleneck basic width is 4, which is divided into 64 group.
```

Comment:
64x4d represents that the bottleneck block has 64 group and each group has basic width of 4